tests/store/message-store.spec.ts

```diff
@@ -161,4 +162,17 @@ describe('MessageStoreLevel Tests', () => {
       expect(results.length).to.equal(0);
     });
   });
+
+  describe('createStore', function () {
+    it('should be called if provided', async () => {
+      new MessageStoreLevel({
+        blockstoreLocation : 'TEST-BLOCKSTORE',
+        indexLocation      : 'TEST-INDEX',
+        createStore(location, options): ReturnType<typeof createStore> {
+          expect(location).to.equal('TEST-BLOCKSTORE');
+          return createStore(location, options);
```

Comment:
Sanity: I think this doesn't actually test if custom `createStore` is called. Will pass regardless it's called or not.

Reply:
oh lol good point ill fix